projects/Mallard/package.json

```diff
@@ -94,6 +94,7 @@
     },
     "devDependencies": {
         "@babel/core": "^7.6.2",
+        "@babel/helper-validator-identifier": "^7.10.4",
```

Comment:
What effect does adding this have?

Reply:
It was not related to the fixing button.

I had an issue building without this. So I added this here. I would imagine it would be a good thing for any new dev too?

Reply:
It is a dev dependency that (for whatever reason is not declared in packages.json file) only exists in yarn.lock file